main.rb

```diff
@@ -0,0 +1,52 @@
+require "terminal-table"
+require_relative "planet"
+require_relative "solar_system"
+
+def main
+  solar_system = SolarSystem.new("A Wrinkle in Time")
+
+    camazotz = Planet.new(“Camazotz”, “red”, 68.9, 22.777, “Ruled by a disembodied brain”)
+    ixchel = Planet.new(“Ixchel”, “pink”, 49, 10.8E9, “Inhabited by sightless creatures”)
+    uriel = Planet.new(“Uriel”, “blue”, 10, 411.5E1, “Lots of extremely tall mountains”)
```

Comment:
Your `main.rb` file had a bunch of weird syntax errors. It somehow ended up with a bunch of angle quotes (`“”`) instead of straight quotes (`"`), which very much confuses the Ruby interpreter. You can see here that the syntax highlighting doesn't understand what's going on.

The only way I've seen this happen is via saving code in a Google doc or word doc or similar application, and then copy/pasting it into your editor directly.

This problem was relatively easy to fix, but isn't one I would expect you to be hitting. I have two asks for you here:
1. Make sure you run your code before submitting it
1. If this comes up and you're not sure how it happened, come find an instructor